code/web/interface/themes/responsive/RecordDrivers/GroupedWork/result.tpl

```diff
@@ -211,12 +211,12 @@
 							{translate text="Formats" isPublicFacing=true}
 						</div>
 						<div class="hidethisdiv{$summId|escape} result-value col-sm-8 col-xs-12">
-							<a onclick="$('#relatedManifestationsValue{$summId|escape},.hidethisdiv{$summId|escape}').toggleClass('hidden-xs');return false;" role="button">
+							<a onclick="$('#relatedManifestationsValue{$summId|escape}').toggleClass('hidden-xs');return false;" role="button">
```

Comment:
Is the removal of 'hidethisdiv' actually necessary? Feels unrelated to this patch to me?